CMakeLists.txt

```diff
@@ -113,9 +113,9 @@ project (Jerry C ASM)
   set(MCU_SCRIPT_GENERATED_HEADER ${CMAKE_BINARY_DIR}/generated.h)
 
 # Should we use external libc?
- if(DEFINED USE_COMPILER_DEFAULT_LIBC AND USE_COMPILER_DEFAULT_LIBC STREQUAL "YES")
+ if(DEFINED COMPILER_DEFAULT_LIBC AND COMPILER_DEFAULT_LIBC STREQUAL "ON")
```

Comment:
Yes, I've changed all occurences of ```USE_COMPILER_DEFAULT_LIBC```.
The ```YES/NO``` -> ```ON/OFF``` modification just simplify the testing-mechanism, because only this option use ```YES/NO```, the others use ```ON/OFF```. 
In my opinion, the ```ON/OFF``` is more obvious without the ```USE_``` prefix.